apps/api/src/auth/auth.service.ts

```diff
@@ -156,12 +156,15 @@ export class AuthService {
         await this.authenticateAdmin()
         const userData = await this.admin.users.findOne({ id: user.sub })
         const registerDto: RegisterDto = {
+          type: ProfileType.INDIVIDUAL,
           email: userData?.email ?? '',
           password: '',
           firstName: userData?.firstName ?? '',
           lastName: userData?.lastName ?? '',
+          companyName: 'typechecking-hack',
```

Comment:
Can we make the field optional, so that we won't have to add artificial data for non-company profiles

Reply:
Technically, those fields are not added anywhere, as calling `createPerson`, doesn't address companyName, and companyNumber as part of the `person` update. Though changing those fields in `RegisterDto` to be `string | undefined`,  seems more appropriate, so will do.

Reply:
Perfect